### PR TITLE
Add CEDS source ID

### LINKED ISSUE #55: Register emissions source ID and institution ID
For the CEDS produced data, we need a source ID and institution ID entry.

Institution ID will be a bit tricky I expect. From what I can tell, the institute is either CEDS or JGCRI. They both appear to be a consortium of PNNL and University of Maryland. Once we have clarity on this, I can help push this upstream to https://github.com/PCMDI/mip-cmor-tables.

In terms of source ID, suggestion below (same idea as https://github.com/PCMDI/input4MIPs_CVs/issues/42)

```json
    "CEDS-YYYY-mm-DD":{
        "contact":"rachel.hoesly@pnnl.gov;ssmith@pnnl.gov;noah.prime@pnnl.gov",
        "further_info_url":"www.tbd.invalid",
        "institution_id":"CEDS",
        "license_id":"CC BY 4.0",
        "mip_era":"CMIP6Plus",
        "source_version":"YYYY.mm.DD"
    }
```

### PR DESCRIPTION
**Pull request by znichollscr**
## Description

Closes #55 

## Checklist

Please confirm that this pull request has done the following:

- [x] Documentation added (where applicable)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

File: CVs/input4MIPs_source_id.json [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+        "source_version":"2024-07-08"
+    },
+    "CEDS-CMIP-2024-07-08-supplemental":{
+        "contact":"Rachel Hoesly (rachel.hoesly@pnnl.gov); Steve Smith (ssmith@pnnl.gov)",
+        "further_info_url":"https://www.pnnl.gov/projects/ceds",
+        "institution_id":"PNNL-JGCRI",
+        "license_id":"CC BY 4.0",
+        "mip_era":"CMIP6Plus",
+        "source_version":"2024-07-08"
+    },
```
Comment (znichollscr) [inline CVs/input4MIPs_source_id.json:17]: @ssmithClimate @hahsan1 can you please confirm these are the entries you want.

Comment (ssmithClimate) [reply CVs/input4MIPs_source_id.json:17]: Looks good to me.

**Comment by znichollscr** [comment]
Alrighty @durack1 over to you for review then I think we can merge

**Review by durack1 — APPROVED** [review]
@znichollscr looks good to me - working out the kinks on the two CEDS source_id's, we can do that after the fact, right?

**Comment by znichollscr** [comment]
Yep, they won't bite until we publish anywa.

**Pull request merged**